benchmarks/piazza/piazza.rs

```diff
@@ -261,7 +280,7 @@ fn main() {
         match benchmark.as_ref() {
             "migration" => {
                 start = time::Instant::now();
-                app.log_user(uid.into(), domain_config);
+                app.log_user(uid.into(), &domain_config);
```

Comment:
If you `#[derive(Clone, Copy)]` for `DomainConfig`, you could leave off the `&`s here, and then `*` for the corresponding `match`es.